### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by thomas-mauran**
## Use chakra ui theme
image

### How should this be manually tested?

- Step 1 : clone and run
- Step 2 : check if theme is present

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "lint": "eslint .",
+    "lint": "prettier --check \"{**/*,*}.{js,ts,jsx,tsx}\"",
```
Comment (mfernd) [inline package.json:9]: problem: there is not enough information (maybe change prettier/eslint by another)

Comment (mfernd) [reply package.json:9]: [Rome](https://rome.tools/) maybe?

**Comment by thomas-mauran** [comment]
LGTM

**Review by mfernd — APPROVED** [review]

**Review by noetarbouriech — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (mfernd) [reply package.json:9]: not important, maybe in the future

**Pull request merged**